setup.py

```diff
@@ -24,11 +27,26 @@ def is_requirement(line):
     """
     return line and not line.startswith(('-r', '#', '-e', 'git+', '-c'))
 
+
 setup(
     name="chem",
-    version='1.0.0',
+    description='openedx chem',
+    long_description=long_description,
+    version='1.1.0',
     packages=["chem"],
     install_requires=load_requirements('requirements/base.in'),
     test_suite='chem.tests',
     tests_require=load_requirements('requirements/test.in'),
+    keywords=['openedx chem'],
+    author='edX',
+    url='https://github.com/edx/openedx-chem',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Build Tools',
```

Comment:
Maybe "Topic :: Software Development :: Libraries" and "Topic :: Scientific/Engineering :: Chemistry" would be better?